docs/framework/unmanaged-api/wmi/beginenumeration.md

```diff
@@ -0,0 +1,107 @@
+---
+title: BeginEnumeration function
```

Comment:
These titles will probably fall in the too short list. We might need to consider appending a value to the title metadata. Also, ideally you'd add the description too.